sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -359,6 +359,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val THRIFTSERVER_RESULT_LIMIT =
+    buildConf("spark.sql.thriftserver.retainedResults")
```

Comment:
In hive, we use parameter "hive.fetch.task.conversion=minimal" to take result from mr job's output(from disk),  in this mode hive will not collect the result to the memory, avoiding the hive process crash. And I can not think of a good name.